dnsapi/dns_linode_v4.sh

```diff
@@ -74,9 +74,9 @@ dns_linode_v4_rm() {
   if _rest GET "/$_domain_id/records" && [ -n "$response" ]; then
     response="$(echo "$response" | tr -d "\n" | tr '{' "|" | sed 's/|/&{/g' | tr "|" "\n")"
 
-    resource="$(echo "$response" | _egrep_o "{.*\"name\":\s*\"$_sub_domain\".*}")"
+    resource="$(echo "$response" | _egrep_o "\{.*\"name\": *\"$_sub_domain\".*}")"
```

Comment:
why bracket `\{` is escaped?

Reply:
It's the only way it worked on the FreeBSD pfSense appliance I was using. I think it has something to do with the POSIX vs PCRE regular expressions implemented in grep for the OS, but that's a little outside of my depth of knowledge. I can only say that it worked only when both the { were escaped AND the \s were converted to hard spaces. Doing one or the other did not result in success.

Reply:
Based on this answer: https://unix.stackexchange.com/a/445641 
It may be best to do [{] rather than escaped for cross-platform compatibility